sycl/test-e2e/CMakeLists.txt

```diff
@@ -0,0 +1,72 @@
+cmake_minimum_required(VERSION 3.15)
+
+message("Configuring SYCL End-to-End Tests")
+
+if(CMAKE_SOURCE_DIR STREQUAL CMAKE_CURRENT_SOURCE_DIR)
+  project(sycl-e2e-test-suite C CXX)
+  set(SYCL_TEST_E2E_STANDALONE TRUE)
+endif()
+
+if(SYCL_TEST_E2E_STANDALONE)
+  if( NOT OpenCL_LIBRARY )
+    find_package(OpenCL)
+  endif()
+endif() # Standalone.
+
+if(SYCL_TEST_E2E_STANDALONE)
```

Comment:
I would suggest to merge this with the previous `if` to make it a bit more concise.

Reply:
I'd prefer to merge as-is for now.